### PR TITLE
Annotate Patterns Examples

### PR DESCRIPTION
**Pull request by RexJaeschke**

### CODE REVIEW

File: standard/patterns.md [hunk 1]
```diff
@@ -160,13 +162,14 @@ A set of patterns `Q` is *exhaustive* for a type `T` if any of the following con
 
 > *Example*:
 >
+> <!-- Example: {template:"standalone-console-without-using", name:"PatternExhaustiveness1", replaceEllipsis:true, customEllipsisReplacements: [""], ignoredWarnings:["CS8321"]} -->
```
Comment (jskeet) [inline standard/patterns.md:165]: Shouldn't we be expecting an error here?

Comment (jskeet) [reply standard/patterns.md:165]: Or rather: fundamentally, the example doesn't fully show what it claims to, due to the "..."
It's not clear to me whether it's worth "testing" something that doesn't behave as documented.

**Review by BillWagner — APPROVED** [review]
I'm good with this.

My thinking on our testing is to validate what we can, while ensuring that the focus is still on the text of the standard, rather than creating a "testable standard".

To that end, let's leave this sample as is. We do verify there are no typos in the text, even though we're not actually testing subsumption.

**Review by jskeet — APPROVED** [review]

**Comment by jskeet** [comment]
Right, I'm fine with that. I'll merge it now.

**Pull request merged**